core/app/models/spree/product/scopes.rb

```diff
@@ -186,9 +186,14 @@ def self.property_conditions(property)
       where("#{Product.quoted_table_name}.deleted_at IS NULL or #{Product.quoted_table_name}.deleted_at >= ?", Time.zone.now)
     end
 
+    def self.currency_available(currency = nil)
+      joins(:master => :prices).where('spree_prices.currency': currency || Spree::Config[:currency]) \
+      .where('spree_prices.amount IS NOT NULL')
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.